content/en/developers/ide_integrations/vscode/_index.md

```diff
@@ -10,25 +10,33 @@ further_reading:
   text: "Learn about the Continuous Testing Tunnel"
+- link: "/integrations/guide/source-code-integration/"
+  tag: "Documentation"
+  text: "Learn about Source Code Integration."
 ---
 
 {{< callout url="#" btn_hidden="true">}}
-  The Datadog extension for Visual Studio Code is in public beta. It is intended for Datadog users who have <a href="https://docs.datadoghq.com/synthetics/#pagetitle">Synthetic tests</a> set up already. The extension supports running <a href="https://docs.datadoghq.com/synthetics/api_tests/http_tests">HTTP tests</a> and <a href="https://docs.datadoghq.com/synthetics/browser_tests">browser tests</a>. If the extensions stops working unexpectedly, check for extension updates or <a href=#feedback>reach out to the team</a>.
+  The Datadog extension for VS Code is in public beta. It is intended for all developers that use Datadog products, and especially those using <a href="https://docs.datadoghq.com/synthetics/#pagetitle">Synthetic tests</a>. If the extension stops working unexpectedly, check for extension updates or <a href=#feedback>reach out to the team</a>.
 {{< /callout >}}
 
 ## Overview
 
-The Datadog extension for Visual Studio Code helps you improve your code reliability by allowing you to [run Synthetic HTTP tests and browser tests on local environments][1] directly in the IDE. The extension ensures that any potential issue resulting from code changes is identified and addressed before the code is deployed into production and impacts your end users.
+The Datadog extension for VS Code integrates with Datadog to accelerate your development.
 
-{{< img src="developers/ide_integrations/vscode/vscode-extension-demo.png" alt="The Datadog Extension in Visual Studio Code" style="width:100%;" >}}
+{{< img src="/developers/ide_integrations/vscode/datadog-vscode.png" alt="The Datadog for VS Code extension" style="width:100%;" >}}
 
-You can use the Datadog extension to:
+The **Code Insights** view keeps you informed about:
+- Issues from [Error Tracking][10]
+- [Vulnerability][11] reports by Application Security Management
+- [Flaky tests][12] detected by CI Visibility
+- Profiling insights from [Watchdog Insights][13]
 
-- Run Synthetic tests on your local environments.
-- Set custom parameters without altering the original test definition.
-- See test results locally in Visual Studio Code and in Datadog to access additional information.
-- Test only what matters by executing relevant tests at the same time.
-- Create a list of most frequently used Synthetic tests by adding them to **Favorites**. 
+The **Synthetic Tests** feature allows you to:
+- Run Synthetic tests on your local environments
+- Set custom parameters without altering the original test definition
+- See test results locally in VS Code and in Datadog to access additional information
+- Test only what matters by executing relevant tests at the same time
+- Create a list of most frequently used Synthetic tests by adding them to **Favorites** 
```

Comment:
Not sure if I would use the word `Favorites` here as we don't have this concept in the IDE anymore.

Reply:
OK, changing to `by linking them to your workspace`